docs/guides/configuration.md

```diff
@@ -178,17 +178,21 @@ ReactOnRails.configure do |config|
   # Change to a value like this example to enable this feature 
   # config.components_subdirectory = "ror_components"
 
-  # Default configuration to auto loading bundles for components.
-  # default is false. The default can be overrident as an option in calls to view helpers
+  # Default is false.
+  # The default can be overidden as an option in calls to view helpers
   # `render_component` and `render_component_hash`. You may set to true to change the default to auto loading.
   config.auto_load_bundle = false
 
-  # default is false
+  # Default is false
   # Set this to true & instead of trying to import the generated server components into your existing
   # server bundle entrypoint, the PacksGenerator will create a server bundle entrypoint using
   # config.server_bundle_js_file for the filename.
   config.make_generated_server_bundle_the_entrypoint = false
 
+  # Default is false
+  # Set this to true to have `defer: false` added to your `append_javascript_pack` calls for generated entrypoints.
```

Comment:
This comment is very confusing.

Default is for `defer` is true or false if not used?

Setting to `true` gives you the default defer?

Reply:
My bad. Corrected.